book/website/community-handbook/templates/template-fireside-chat.md

```diff
@@ -52,7 +52,7 @@ Please propose your suggestion below or reach out to *The Turing Way* team membe
   * Turn on your webcam if you don’t mind sharing your face (or off if you do!)
 * Reminder: 
     * **Code of conduct**: [https://book.the-turing-way.org/community-handbook/coc.html](https://book.the-turing-way.org/community-handbook/coc.html)
-       * If you experience or witness unacceptable behaviour, or have any other concerns, please report it by contacting the project members - Malvika and Kirstie Whitaker (turingway@gmail.com).
+       * If you experience or witness unacceptable behaviour, or have any other concerns, please report it by contacting the project delivery team (turingway@gmail.com).
```

Comment:
```suggestion
       * If you experience or witness unacceptable behaviour, or have any other concerns, please report it by contacting the project delivery team (theturingway@gmail.com).
```

Reply:
It might be helpful to have an internal link to the Project delivery team in the book and also detail who to go to if anything needs to be raised _about_ someone or some people from the Project delivery team. I wrote something for the NHS-R Community to say who to go to if the concern is about me for example: https://github.com/nhs-r-community/NHSR-way/blob/f22709d2cf27af532ac98534f94f202dd9cec277/code-of-conduct.qmd#L44 as internal structures of responsibility are not always obvious.

Reply:
Hi @Lextuga007, this PR is creating a subpage with the info about project team which currently only is in the base of GitHub repo.

@aleesteele -- this PR is maintaining turing email. We will change in once we have clarity on when we need to move the infra.